.github/workflows/release.yaml

```diff
@@ -46,15 +46,15 @@ jobs:
 
       - name: Checkout dist repo
         uses: actions/checkout@v2
-        if: steps.changesets.outputs.published
+        if: steps.changesets.outputs.published && steps.changesets.outputs.published != 'false'
```

Comment:
Since this is a string, do we want to do `== 'true'` instead? I'd rather be defensive against values other than `'true'` and `'false'` and avoid the double negative of "not false".

Reply:
Yeah, that makes more sense. Updated.

Reply:
Oops, I think this was my suggestion -- I thought that `published` was either the list of published packages, or `false`. But actually it probably is `"true"` or `"false"`, and the separate `publishedPackages` output is the list of packages. So just checking `"true"` sounds good to me too.

https://github.com/changesets/action#outputs